src/windshaft/map-serializer/anonymous-map-serializer/layers-serializer.js

```diff
@@ -77,6 +77,15 @@ function optionsForMapnikLayer (layerModel) {
     options.aggregation = aggregation;
   }
 
+  var minzoom = layerModel.get('minzoom');
+  if (Number.isFinite(minzoom)) {
+    options.minzoom = minzoom;
+  }
+  var maxzoom = layerModel.get('maxzoom');
+  if (Number.isFinite(maxzoom)) {
```

Comment:
this wont work in IE :trollface:

Reply:
I know, but you are here to help me with the right approach. 😄